README.md

```diff
@@ -67,7 +71,7 @@ For example, change the referenced image in [springboot](https://github.com/ziqu
 
 ## Deploy Things
 
-Deploy the postgres container (this needs to happen first so the service host/port envs properly load into subsequent containers)
+Deploy the postgres container (this needs to happen first so the service host/port envs properly load into subsequent containers) (@z - on my GKE cluster, built from the standard template, I have to `cd kubernetes_datadog` before running - is that folder loaded automatically? Was this because I took your workshop before? I'm using a new cluster, so it's not clear to me how this gets folder loaded. Regardless, the following commands require the user be in the `kubernetes_datadog` folder)
```

Comment:
I think that is the expectation (that you'd need to be in the folder where the `*.yaml` exists or point to that path, e.g. `kubectl create -f ~/path/to/kubernetes_datadog/postgres_deployment.yaml` -- perhaps simply adding a step here stating you should change into the project directory would suffice.